driver/razerkbd_driver.c

```diff
@@ -1533,7 +1549,11 @@ static ssize_t razer_attr_write_matrix_effect_reactive(struct device *dev, struc
     case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_PRO_WIRED:
         report = razer_chroma_extended_matrix_effect_reactive(VARSTORE, BACKLIGHT_LED, speed, (struct razer_rgb*)&buf[1]);
         break;
-
+    case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_PRO_WIRELESS:
+        razer_set_device_mode(usb_dev, 0x00, 0x00);
```

Comment:
If the daemon likes to switch the device to driver mode (which I didn't check now), then maybe that should be removed if reactive effect doesn't work with it.
Overwriting the device mode here isn't great at all...